src/styles/main.scss

```diff
@@ -1,3 +1,77 @@
+@import "../styles/utils/var.scss";
+@import "../styles/mixin/mixin.scss";
+
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 30px;
+}
+
 body {
   margin: 0;
+  width: 100%;
+}
+
+.calendar {
+  display: flex;
+  flex-direction: column;
+  gap: $cellGap;
+  padding: $padd;
+  max-width: $weekWidth;
+  justify-content: center;
+  margin-left: 25%;
+  margin-top: 10%;
+}
+
+.week {
+  display: flex;
+  flex-direction: row;
+  gap: $cellGap;
+  flex-wrap: wrap;
+
+  &__days {
+    @include square($size);
+    border: 1px solid black;
+    background-color: $calendColor;
+    text-align: center;
+    align-items: center;
+    justify-content: center;
+    position: relative;
+    transition: transform 0.5s;
+  }
+
+  &__days:hover {
+    background-color: pink;
+    cursor: pointer;
+    transform: translateY(-20px);
+  }
+
+  &__days::before {
+    content: "";
+    position: absolute;
+    top: 4vh;
+    left: 3vw;
```

Comment:
remove it if you center content via flex
and remove this selector at all since you are setting content below